addons/mail/static/src/messaging/component/activity/activity.xml

```diff
@@ -112,15 +112,14 @@
                     <t t-if="activity.canWrite">
                         <div class="o_Activity_tools">
                             <t t-if="activity.category !== 'upload_file'">
-                                <ActivityMarkDoneButton
-                                    class="o_Activity_toolButton o_Activity_markDoneButton btn-link"
-                                    activityLocalId="activity.localId"
-                                    popoverTitle="'Mark Done'"
-                                >
-                                    <t t-set="buttonHtml">
+                                <Popover position="'right'" title="popoverTitle">
```

Comment:
Can't use translatable string literal in template, which is why you use getter `popoverTitle`.
Instead of naming it `popoverTitle`, I suggest something like `MARK_DONE`, so that variable name almost match string. This makes reading the template easier, and almost look like the intended way of having string literal just here.

Reply:
ok